[NEAT-218] 🐻‍❄️ Ensure Lifecycle Tutorials are up-to-date (#457)
cognitedata/neat: docs/tutorials/data-modeling-lifecycle/part-3-extending-enterprise-model.md

  (see definition of headings [here](../../terminology/rules.md#properties-sheet))
* **Classes**: This contains the classes for the changes, and will only have headings
  (see definition of headings [here](../../terminology/rules.md#classes-sheet))
* **LastProperties**: (READ-ONLY) This will be all the properties from the enterprise model that Svein Harald can use to
  look up what properties he wants to use in the solution model. In addition, this will be used in the validation
  to ensure that the new changes do break the existing model.
* **LastClasses**: (READ-ONLY) This will be all the classes from the current enterprise model. Similar to the `LastProperties`,
  this will be used to look up, and will be validated against.
* **LastMetadata**: (READ-ONLY) This will be the metadata from the current Enterprise model.

> **Node** The **Last** sheets are used by **NEAT** to validate against depending on the `extension` in the `Metadata`

Reviewer: ```suggestion
**Note** The **Last** sheets are used by **NEAT** for validations, which are dependent the `extension` configuration in the `Metadata`
```
